code/game/machinery/cryopod.dm

```diff
@@ -335,7 +339,7 @@
 
 		var/preserve = null
 		for(var/T in preserve_items)
-			if(istype(W,T))
+			if(istype(W,T) && !(W in do_not_preserve_items))
```

Comment:
There is probably a better way to do this, given that any posibrains on people will just get unceremoniously qdeled